### DISCUSSION TITLE
Images preprocessing for transfer learning of yolov8

### DISCUSSION THREAD

**Discussion by mgeorgie95** [ultralytics/yolov5#13275] [Q&A]
hello there, 
kindly, can you help me? I have tried a lot to make the preprocessing of Downloaded CottonWeedDet12 for transfer learning of yolov8 variants, but unfortunately, I am getting nothing, what can  I do? , below is the output from python console:

C:\Users\acer\.conda\envs\pytorch\python.exe "C:\Users\acer\PycharmProjects\pythonProject5\Images preprocessing.py"
Loaded labels for 6 images from C:\Users\acer\PycharmProjects\pythonProject5\Weed12dataset\dataset\weedcoco.json
Total images loaded: 0
Total labels loaded: 0
Train set size: 0
Validation set size: 0
Test set size: 0
No images to process in C:\Users\acer\PycharmProjects\pythonProject5\train
No images to process in C:\Users\acer\PycharmProjects\pythonProject5\val
No images to process in C:\Users\acer\PycharmProjects\pythonProject5\test
Processing complete.

Process finished with exit code 0
